action/candidate_endorsement.go

```diff
@@ -35,12 +35,61 @@ const (
 			"outputs": [],
 			"stateMutability": "nonpayable",
 			"type": "function"
+		},
+		{
+			"inputs": [
+				{
+					"internalType": "uint64",
+					"name": "bucketIndex",
+					"type": "uint64"
+				}
+			],
+			"name": "candidateEndorse",
+			"outputs": [],
+			"stateMutability": "nonpayable",
+			"type": "function"
+		},
+		{
+			"inputs": [
+				{
+					"internalType": "uint64",
+					"name": "bucketIndex",
+					"type": "uint64"
+				}
+			],
+			"name": "candidateUnEndorse",
+			"outputs": [],
+			"stateMutability": "nonpayable",
+			"type": "function"
+		},
+		{
+			"inputs": [
+				{
+					"internalType": "uint64",
+					"name": "bucketIndex",
+					"type": "uint64"
+				}
+			],
+			"name": "candidateUnEndorseSettle",
+			"outputs": [],
+			"stateMutability": "nonpayable",
+			"type": "function"
 		}
 	]`
 )
 
+// CandidateEndorsementOpt defines the operation of CandidateEndorsement
+const (
+	CandidateEndorsementOpEndorse = iota + 1
+	CandidateEndorsementOpUnendorse
```

Comment:
better to explicitly give the value of const

Reply:
can you talk more about the reason? I think it's more concise, without specify the value one by one